README.md

```diff
@@ -24,16 +24,22 @@ Or install it yourself as:
 
 ## Usage
 
-```ruby
-$ iconv -f cp932 -t utf-8 < alert-201912.txt | bundle exec rake waffle:filter
+By passing the WAF log from the standard input, the log that matches the route is written to the standard output.
+
+```sh
+$ bundle exec rake waffle:filter < alert-201912.txt
 ```
 
-with options
+If the character encoding of the log is CP932, specify the encoding with the environment variable `RUBYOPT`.
 
-```ruby
-$ iconv -f cp932 -t utf-8 < alert-201912.txt | bundle exec rake waffle:filter -f 2 -w 2
+```sh
+$ env RUBYOPT='-ECP932:CP932' bundle exec rake waffle:filter < alert-201912.txt
```

Comment:
Thanks for updating the README.

However, RUBYOPT does not seem to work as expected.
Please switch back this line back to use iconv.